docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -0,0 +1,169 @@
+---
+id: index
+title: "Zeebe Gateway"
+sidebar_label: "Overview"
+description: "The Zeebe Gateway is a component of the Zeebe cluster."
+---
+
+## What is the Zeebe Gateway?
+
+The Zeebe Gateway is a component of the Zeebe cluster; it can be considered the contact point for the Zeebe cluster which allows Zeebe clients to communicate with Zeebe brokers inside a Zeebe cluster. For more information about the Zeebe broker, visit our [additional documentation](../../components/zeebe/technical-concepts/architecture.md#brokers).
+
+To summarize, the Zeebe broker is the main part of the Zeebe cluster, which does all the heavy work like processing, replicating, exporting, and everything based on partitions. The Zeebe Gateway acts as a load balancer and router between Zeebe’s processing partitions.
+
+![Zeebe gateway overview](assets/zeebe-gateway-overview.png)
+
+To interact with the Zeebe cluster, the Zeebe client sends a command as a gRPC message to the Zeebe Gateway (to port `26500` by default). Given the gateway supports gRPC, the user can use several clients in different languages to interact with the Zeebe cluster. For more information, read our [overview](../../apis-clients/overview.md).
+
+When the Zeebe Gateway receives a valid gRPC message, it is translated to an internal binary format and forwarded to one of the partition leaders inside the Zeebe cluster. The command type and values can determine to which partition the command is forwarded.
+
+For example, creating a new process instance is sent in a round-robin fashion to the different partitions. If the command relates to an existing process instance, the command must be sent to the same partition where it was first created (determined by the key).
+
+To determine the current leader for the corresponding partition, the gateway must maintain the topology of the Zeebe cluster. The gateway(s) and broker(s) form a cluster using gossip to distribute information.
```

Comment:
What do you mean by "topology" and "gossip" here?

Reply:
"Topology" is I would say a common term in distributed systems. The topology describes the number of nodes in a cluster and their state (leader/follower) etc.

With gossip I referring to the Gossip protocol, sorry I probably should have added a link to it. https://en.wikipedia.org/wiki/Gossip_protocol#:~:text=A%20gossip%20protocol%20or%20epidemic,all%20members%20of%20a%20group.

Reply:
Excellent! I added "protocol" to gossip in my latest commit.